packages/react-button/src/components/MenuButton/stories/MenuButtonDisabled.stories.tsx

```diff
@@ -20,7 +20,7 @@ export const Disabled = () => {
       </Menu>
       <Menu>
         <MenuTrigger>
-          <MenuButton disabled>This is disabled</MenuButton>
+          <MenuButton disabled>Disabled</MenuButton>
```

Comment:
Maybe this could be "Disabled state" (and also Enabled state?). Just thinking that "disabled button disabled" is a little bit of a weird aural UI

```suggestion
          <MenuButton disabled>Disabled State</MenuButton>
```